docs/buildah-config.md

```diff
@@ -13,24 +13,24 @@ Updates one or more of the settings kept for a container.
 
 **--annotation** *annotation*
 
-Adds an image *annotation* (e.g. annotation=*annotation*) to the image manifest
-of any images which will be built using the specified container.
+Add an image *annotation* (e.g. annotation=*annotation*) to the image manifest
```

Comment:
could you whip up an example or three that shows how some of the multiples could be used?  Perhaps an example for annotations, port and volume?